app/jobs/application_submission_job.rb

```diff
@@ -1,88 +0,0 @@
-class ApplicationSubmissionJob < ApplicationJob
```

Comment:
Thos are in Leandro's pr, can we rebase off his branch otherwise we're stepping on each other's toes?

Reply:
Sure, will rebase after Leandro's PR